ansible_collections/arista/cvp/plugins/module_utils/container_tools.py

```diff
@@ -848,7 +848,7 @@ def configlets_detach(self, container: str, configlets: List[str]):
         container_info = self.get_container_info(container_name=container)
         detach_configlets = list()
         for configlet in configlets:
-            data = self.__get_configlet_info(configlet_name=configlet['name'])
+            data = self.__get_configlet_info(configlet_name=configlet)
```

Comment:
Under investigation as it may break execution

Reply:
@titom73 : Yes, this change is breaking the execution. No need of this change. I have back out this change and made changes in system test case accordingly.